contracts/interchain-token-service/src/contract/execute.rs

```diff
@@ -196,6 +197,49 @@ pub fn set_chain_config(
     }
 }
 
+#[allow(unused)]
+fn translate_and_save_token_decimals_on_token_deployment(
+    storage: &mut dyn Storage,
+    src_chain: &ChainNameRaw,
+    dest_chain: &ChainNameRaw,
```

Comment:
```suggestion
    source_chain: &ChainNameRaw,
    destination_chain: &ChainNameRaw,
```

Reply:
I was hopping to get shorter lines, but sure.